src/org/usfirst/frc/team5968/robot/Autonomous.java

```diff
@@ -0,0 +1,86 @@
+package org.usfirst.frc.team5968.robot;
+
+import edu.wpi.first.wpilibj.DriverStation;
+import edu.wpi.first.wpilibj.DriverStation.Alliance;
+
+public class Autonomous implements IAutonomous{
+	 /*
+     * Should be called in a fast loop (~20ms) while disabled to get data about 
+     * switch/scale sides. Returns a string of length 3 detailing
+     * the alliance sides, starting from the closest (own) switch.
+     * Ex. "LRL" "RRL" "RRR"
+     */
+	private String pollGameData() {
+		String gameData;
+		boolean status = DriverStation.getInstance().isDisabled();
```

Comment:
`status = true` doesn't tell us whether the robot is disabled or not. I would rename `status` to something along the lines of `isRobotDisabled`